### PR TITLE
Modified record_statistics.html file

### PR DESCRIPTION
**Pull request by maheshkkumar**

### CODE REVIEW

File: feedback-front/client.py [hunk 1]
```diff
@@ -17,6 +18,15 @@ def __init__(self):
     def fetch_dataset_statistics(self):
         return requests.get(STATS_RESOURCE).json()["datasetStatistics"]
 
+    def fetch_dataset_records(self, dataset):
```
Comment (samarthbhargav) [inline feedback-front/client.py]: fetch_record_statistics

File: feedback-front/static/js/charts.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    d3.select(chartDiv+ ' svg')
+        .datum(data)
+        .transition().duration(350)
+        .call(chart);
+
+  return chart;
+});
+}
+
+$(document).ready(function(){
```
Comment (samarthbhargav) [inline feedback-front/static/js/charts.js]: Will every html page that includes this js file will have a chart rendered? 
Should this not be put into files that actually needs a chart rendered?

File: feedback-front/static/js/histogram.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ function renderBarChart(data, chartDiv){
                 .x(function(d) { return d.label })
```
Comment (samarthbhargav) [inline feedback-front/static/js/histogram.js]: Why are there two files doing the same thing?

File: feedback-front/templates/record_statistics.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+<script>
+  
+  // rendering pie chart
+  var data = [{
+    key : "Dataset stats",
+    values : [
+      {% for stat in stats %}
+          {
+              label : "{{ stat.label}}",
```
Comment (samarthbhargav) [inline feedback-front/templates/record_statistics.html]: Where is the if else part?

File: feedback-front/templates/record_statistics.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+<div id="Div2">
+  <div id="chart2">
+    <svg></svg>
+  </div>
+</div>  
+
+<!-- Javascript for toggling the charts -->
+<script>
+function switchCharts() {
+  if (document.getElementById('Div1')) {
```
Comment (samarthbhargav) [inline feedback-front/templates/record_statistics.html]: Don't use generic div names. Name it something that is readable - "barChartDiv" or something.

File: feedback-front/client.py (hunk 1 above)
Comment (maheshkkumar) [inline feedback-front/client.py]: Yeah, fixed it

File: feedback-front/static/js/charts.js (hunk 2 above)
Comment (maheshkkumar) [inline feedback-front/static/js/charts.js]: No, not every file will need it, if for instance we require either bar chart or pie chart, then we can include this into that, that's the reason I created a seperate .js file.

File: feedback-front/static/js/histogram.js (hunk 3 above)
Comment (maheshkkumar) [inline feedback-front/static/js/histogram.js]: Yeah, will remove histogram.js file

File: feedback-front/templates/record_statistics.html (hunk 5 above)
Comment (maheshkkumar) [inline feedback-front/templates/record_statistics.html]: Yeah, will work on that

File: feedback-front/templates/record_statistics.html (hunk 4 above)
Comment (maheshkkumar) [inline feedback-front/templates/record_statistics.html]: Had little issue with that part, let me work on that

File: feedback-front/templates/record_statistics.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  if (document.getElementById('barChart')) {
+    if (document.getElementById('barChart').style.display == 'none') {
+        document.getElementById('barChart').style.display = 'block';
+        document.getElementById('pieChart').style.display = 'none';
+        document.getElementById('Button1').innerHTML = "Click Here Pie Chart";
+    }
+    else {
+        document.getElementById('barChart').style.display = 'none';
+        document.getElementById('pieChart').style.display = 'block';
+        document.getElementById('Button1').innerHTML = "Click Here Bar Chart";
```
Comment (samarthbhargav) [inline feedback-front/templates/record_statistics.html]: Button1?

File: feedback-front/templates/record_statistics.html [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+  // rendering bar chart
+  var data = [
+        {% for stat in stats %}
+          {
+            label : "{{ stat.label}}",
+            value : {{ stat.numberOfRecords }}
+          },
+        {% endfor %}
+    ];
+    renderPieChart(data, "#chart2");
```
Comment (samarthbhargav) [inline feedback-front/templates/record_statistics.html]: descriptive names

File: feedback-front/static/js/charts.js (hunk 2 above)
Comment (samarthbhargav) [inline feedback-front/static/js/charts.js]: Dude - remove this. Put it only pages that require it.

**Pull request merged**